index.coffee

```diff
@@ -1,6 +1,8 @@
+GameGrid    = require ("./app/models/GameGrid")
 InputSolver = require ("./app/services/InputSolver")
-GameGrid    = require ("./app/GameGrid")
+ExpressionGenerator = require("./app/services/ExpressionGenerator");
 
-
-console.log InputSolver.compute("1+2*-3")
-console.log new GameGrid(3)
+generator = new ExpressionGenerator()
+for i in [1...30]
```

Comment:
I know this is just testing but should we keep using ```[1..30]``` (two dots) for consistency?